map/mwm_url.cpp

```diff
@@ -324,7 +324,10 @@ bool ParsedMapApi::RouteKeyValue(string const & key, string const & value, vecto
 {
   using namespace route;
 
-  if (pattern.empty() || key != pattern.front())
+  if (pattern.empty())
```

Comment:
just to make sure we are on the same page, are we support deep links for routing only?

Reply:
*back_url for routing deep links only

Reply:
Yes, it is. I would like to expand support of back url in deeplink gradually, by feature request. In same time, I've added this support for all mapsme:// deeplinks already and will ask to QA to test the rest mapsme:// deeplinks, with and without back_url parameter.

Reply:
Hi @milchakov,
From our app, we launch MAPSME in three ways and would like to be able to go back into our app using the escape key.  The idea is we can go back and forth between the two apps using hardware controls.

* Launch/Bring Forward MAPSME without interrupting any current routing
* Launch MAPSME to route to waypoint
* Launch MAPSME to display waypoint